Tasks/QuickPerfTest/task.json

```diff
@@ -3,7 +3,7 @@
   "name": "QuickPerfTest",
   "friendlyName": "Cloud-based Web Performance Test",
```

Comment:
I'll need to ask someone who is better informed about this task to author the Readme.